R/FIFA2018.R

```diff
@@ -0,0 +1,104 @@
+#'
+#' @usage data("FIFA2018", package = "distributions3")
+#'
+#' @format A data frame with 128 rows and 7 columns.
+#' \describe{
+#'   \item{goals}{integer. Number of goals scored in normal time (90 minutes), \
+#'     i.e., excluding potential extra time or penalties in knockout matches.}
+#'   \item{team}{character. 3-letter FIFA code for the team.}
+#'   \item{match}{integer. Match ID ranging from 1 (opening match) to 64 (final).}
+#'   \item{type}{factor. Type of match for groups A to H, round of 16 (R16), quarter final,
+#'     semi-final, match for 3rd place, and final.}
+#'   \item{stage}{factor. Group vs. knockout tournament stage.}
+#'   \item{logability}{numeric. Estimated log-ability for each team based on
+#'     bookmaker consensus model.}
+#'   \item{difference}{numeric. Difference in estimated log-abilities between
+#'     a team and its opponent in each match.}
+#'   }
+#'
+#' @source The goals for each match have been obtained from Wikipedia
+#'   (\url{https://en.wikipedia.org/wiki/2018_FIFA_World_Cup}) and the log-abilities
+#'   from Zeileis et al. (2018) based on quoted odds from Oddschecker.com and Bwin.com.
+#'
+#' @references Leitner C, Zeileis A, Hornik K (2010).
+#'   Forecasting Sports Tournaments by Ratings of (Prob)abilities: A Comparison for the EURO 2008.
+#'   \emph{International Journal of Forecasting}, \bold{26}(3), 471-481.
+#'   \doi{10.1016/j.ijforecast.2009.10.001}
+#'
+#' Zeileis A, Leitner C, Hornik K (2018).
+#'   Probabilistic Forecasts for the 2018 FIFA World Cup Based on the Bookmaker Consensus Model.
+#'   Working Paper 2018-09, Working Papers in Economics and Statistics,
+#'   Research Platform Empirical and Experimental Economics, University of Innsbruck. 
+#'   \url{https://EconPapers.RePEc.org/RePEc:inn:wpaper:2018-09}
+#'
+#' @examples
+#' ## load data
+#' data("FIFA2018", package = "distributions3")
+#'
+#' ## observed relative frequencies of goals in all matches
+#' obsrvd <- proportions(table(FIFA2018$goals))
+#'
+#' ## expected probabilities assuming a simple Poisson model
+#' ## with constant average number of goals across all teams/matches
+#' p_const <- Poisson(lambda = mean(FIFA2018$goals))
+#' p_const
+#' expctd <- pdf(p_const, 0:6)
+#' 
+#' ## comparison: observed vs. expected frequencies
+#' ## frequencies for 3 and 4 goals are slightly overfitted
+#' ## while 5 and 6 goals are slightly underfitted
+#' cbind("observed" = obsrvd, "expected" = expctd)
+#'
+#' ## instead of fitting the same average Poisson model to all
+#' ## teams/matches, take ability differences into account
+#' m <- glm(goals ~ difference, data = FIFA2018, family = poisson)
+#' summary(m)
+#' ## when the ratio of abilities increases by 1 percent, the
+#' ## expected number of goals increases by around 0.4 percent
+#'
+#' ## this yields a different predicted Poisson distribution for
+#' ## each team/match
+#' p_reg <- Poisson(lambda = fitted(m))
```

Comment:
I think students might confuse confidence intervals and prediction intervals here. Would you be willing to add some quick comments about the exact estimand of the distributional result?

Reply:
Thanks for the pointer, I extended the comment in the example somewhat to be more precise about what is computed here.